openhub/management/commands/import_outside_committers_data.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import json
+import xmltodict
+import logging
+
+from openhub.models import OutsideCommitter, ContributionsToPortfolioProject
+from django.core.management.base import BaseCommand
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+org_name = get_owner()
+class Command(BaseCommand):
+    def import_data(self, contributor):
```

Comment:
this doesnt use `self`.  It shouldnt be in the class.

Reply:
@jayvdb, when I am removing the `self` from the `import_data()` method, I am getting this error:
```
TypeError: import_data() takes 1 positional argument but 2 were given
```

Reply:
Got it; I have to use `@staticmethod` decorator :+1: